processing/src/main/java/io/druid/query/filter/Filter.java

```diff
@@ -27,4 +27,12 @@
 {
   public ImmutableBitmap getBitmapIndex(BitmapIndexSelector selector);
   public ValueMatcher makeMatcher(ValueMatcherFactory factory);
+
+  public ValueMatcher makeMatcher(
```

Comment:
It'd be nice to have a comment about how a Filter should behave when these methods called. Put yourself in the mindset of someone implementing a Filter as an extension; what do they have to do?

I feel like `getBitmapIndex` and the old `makeMatcher` are pretty self-explanatory but the new ones are less so.